card/db/migrate_core_cards/20150611203506_rails_inflection_updates.rb

```diff
@@ -0,0 +1,55 @@
+# -*- encoding : utf-8 -*-
+class RailsInflectionUpdates < Card::CoreMigration
+  def word ar
+    return [/(?<=\b|_)#{ar[0]}(?=\b|_)/i, /(?<=\b|_)#{ar[1]}(?=\b|_)/i, ar[2]]
+  end
+  def word_end ar
+    return [/#{ar[0]}(?=\b|_)/i, /#{ar[1]}(?=\b|_)/i, ar[2]]
+  end
+
+  def keep_the_s word
+    return [ "#{word}s", word, "#{word}s" ]
+  end
+
+  def up
+    card_names = Card.pluck(:name)
+    apply_to_content = ::Set.new
+    corrections =   [
+             # plural,     wrong singular,  correct singular
+      word([ '(\w+)lice',  '(\w+)louse',    '\1lice' ]),
+      word([ '(\w+)mice',  '(\w+)mouse',    '\1mice' ]),
+      word([ 'kine',       'cow',           'kine']),
```

Comment:
Rails 4 dropped support for more than one cow :) And the plural of Moscow is no longer Moskine.